hostprocess/flannel/kube-proxy/start.ps1

```diff
@@ -61,7 +61,7 @@ Write-Host "sourceip: $vip"
 
 $arguements = "--v=6",
         "--hostname-override=$env:NODE_NAME",
-        "--feature-gates=WinOverlay=true,IPv6DualStack=false",
+        "--feature-gates=WinOverlay=true",
```

Comment:
WinOverlay is beta / defaults to try since 1.20.
Should we just drop this line?